src/components/Navbar/LogoWithText.tsx

```diff
@@ -45,7 +46,9 @@ const LogoWithText = () => {
     >
       <img className={classes.logo} src={TonLogo} alt="" />
       <StyledText>
-        <strong>Ton</strong>Swap
+        <strong>Ton</strong>Swap{' '}
```

Comment:
Consider moving this is some util method

Reply:
moved